src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -69,10 +71,10 @@ object EasyStageDataset {
       _ <- writeFoxml(sdoDir, foxmlContent)
       _ <- writePrsql(sdoDir, PRSQL.create())
       _ <- writeJsonCfg(sdoDir, jsonCfgContent)
-    } yield ()
+    } yield emdContent
   }
 
-  private def createFileAndFolderSdos(dir: File, parentSDO: String)(implicit s: Settings): Try[Unit] = {
+  private def createFileAndFolderSdos(dir: File, parentSDO: String)(implicit s: Settings, rights: AccessCategory): Try[Unit] = {
```

Comment:
Personally I would prefer this access category to not be implicit. It would be clearer to have this as a 'normal' parameter, especially as the access category is not a singleton on project level (like Settings is).